src/main.cpp

```diff
@@ -5996,7 +6009,7 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
     }
 
 
-    else if (strCommand == NetMsgType::INV)
+    else if (strCommand == NetMsgType::INV && !fImporting && !fReindex)
```

Comment:
the !fImporting and !fReindex here and in the other places need to be in a separate if statement below this one.  Otherwise the code will fall into the final "else" logic that is meant to be called only if the strCommand is unknown.

Reply:
fixed.